root/application/src/test/java/at/fhv/itb5c/application/ApplicationFacadeTest.java

```diff
@@ -19,244 +19,284 @@
-		assertNotNull(retrievedDTO);
-		assertEquals(id, retrievedDTO.getId());
-	}
-
-	@Test
-	public void testGetDepartmentByIdNull() {
-		DepartmentDTO dto = _appFacade.getDepartmentById(_session, null);
-		assertNull(dto);
-	}
-
-	@Test
-	public void testGetAllDepartments() {
-		final Long id = 1337l;
-		DepartmentDTO dto = ConverterDepartmentDTO.toDTO(new Department());
-		dto.setId(id);
-
-		DepartmentDTO savedDTO = _appFacade.saveDepartment(_session, dto);
-		assertNotNull(savedDTO);
-		assertEquals(id, savedDTO.getId());
-
-		Collection<DepartmentDTO> dtos = _appFacade.getAllDepartments(_session);
-		assertNotNull(dtos);
-		assertTrue(dtos.size() > 0);
-	}
-
-	@Test
-	public void testCreateTeam() {
-		TeamDTO dto = _appFacade.createTeam(_session);
-		assertNotNull(dto);
-		assertNotNull(dto.getMemberIds());
-	}
-
-	@Test
-	public void testFindTeamsAllNull() {
-		Collection<TeamDTO> dtos = _appFacade.findTeams(_session, null, null, null, null);
-		assertNotNull(dtos);
-		assertEquals(0, dtos.size());
-	}
-
-	@Test
-	public void testSaveTeam() {
-		final Long id = 1337l;
-		TeamDTO dto = _appFacade.createTeam(_session);
-		dto.setId(id);
-
-		TeamDTO savedDTO = _appFacade.saveTeam(_session, dto);
-		assertNotNull(savedDTO);
-		assertEquals(id, savedDTO.getId());
-		assertFalse(dto == savedDTO);
-	}
-
-	@Test
-	public void testSaveTeamNull() {
-		TeamDTO dto = _appFacade.saveTeam(_session, null);
-		assertNull(dto);
-	}
-
-	@Test
-	public void testGetTeamById() {
-		final Long id = 1337l;
-		TeamDTO dto = _appFacade.createTeam(_session);
-		dto.setId(id);
-
-		TeamDTO savedDTO = _appFacade.saveTeam(_session, dto);
-		assertNotNull(savedDTO);
-		assertEquals(id, savedDTO.getId());
-		assertFalse(dto == savedDTO);
-
-		TeamDTO retrievedDTO = _appFacade.getTeamById(_session, id);
-		assertNotNull(retrievedDTO);
-		assertEquals(id, retrievedDTO.getId());
-		assertFalse(savedDTO == retrievedDTO);
-	}
-
-	@Test
-	public void testGetTeamByIdNull() {
-		TeamDTO dto = _appFacade.getTeamById(_session, null);
-		assertNull(dto);
-	}
-
-	@Test
-	public void getLeagueByIdNull() {
-		LeagueDTO dto = _appFacade.getLeagueById(_session, null);
-		assertNull(dto);
-	}
-
-	@Test
-	public void getAllLeagues() {
-		Collection<LeagueDTO> dtos = _appFacade.getAllLeagues(_session);
-		assertNotNull(dtos);
-	}
-
-	@Test
-	public void createNewTournament(){
-		DepartmentDTO dept = new DepartmentDTO();
-		dept.setHeadId(_user.getId());
-		TournamentDTO dto = _appFacade.createTournament(_session, dept);
-		assertNotNull(dto);
-		
-		UserDTO standardUser = new UserDTO();
-		standardUser.setId(3l);
-		standardUser.setRoles(new HashSet<>(Arrays.asList(UserRole.StandardUser)));
-		String session = SessionManager.getInstance().createNewSession(standardUser.getId(), standardUser.getRoles());
-		dept.setHeadId(standardUser.getId()+1);
-		dto = _appFacade.createTournament(session, dept);
-		assertNull(dto);
-	}
+//	ApplicationFacade _appFacade;
```

Comment:
These (old) tests don't only use fixed IDs, but also are dependent on each other. -> Depending on the order of execution, the whole test class might or might not fail. Basically, all of them need to be rewritten.